hardhat.config.js

```diff
@@ -109,21 +125,20 @@ module.exports = {
       allowUnlimitedContractSize: true,
       timeout: 100000,
     },
-    mainnet: {
-      url: process.env.ALCHEMY_NODE,
-      accounts: [process.env.PRIV_KEY_MAINNET],
-      gasPrice: 40000000000,
-    },
-    rinkeby: {
-      url: process.env.ALCHEMY_NODE_RINKEBY,
-      accounts: [process.env.PRIV_KEY_MAINNET],
-      gasPrice: 40000000000,
-    },
-    goerli: {
-      url: process.env.ALCHEMY_NODE_GOERLI,
-      accounts: [process.env.PRIV_KEY_MAINNET],
-      gasPrice: 40000000000,
-    },
+    ...[
```

Comment:
What is purpose of this ? I would argue if that is more readable even if shorter

Reply:
i.e. - i don't have the goerli and rinkeby rpc URLs set and the code before required all to be provided no matter what kind of test you are running

Reply:
so basically this protects against someone having incomplete .env file ? I'm not sure if that is something we should protect against I would argue that it is a feature not a bug that hardhat is failing if you do not have all env's set properly.